templates/shared/_contact-us-form.html

```diff
@@ -49,7 +49,7 @@ <h3>問い合わせ欄</h3> <!-- Your comments -->
           <ul class="p-list">
             <li class="p-list__item">
               <label for="Comments_from_lead__c">ご質問及び依頼内容を記載ください。</label> <!-- What would you like to talk to us about? -->
-              <textarea required="" id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" maxlength="2000" aria-required="true"></textarea>
+              <textarea required="" id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" maxlength="2000"></textarea>
```

Comment:
Just checked and this field is not required:
![image](https://user-images.githubusercontent.com/1413534/150105641-f1b52a23-0e7e-4bde-a326-bf61e48b9a26.png)